deferral/ah-django
# #164859737 Upgrade to  django2.1.7

#### What does this PR do?
- Upgrade Django to v2.1.7

#### Description of Task to be completed?
 - [X] ensure the codebase uses the latest django features for python3

#### How should this be manually tested?
- Access the repository on github from [here](https://github.com/andela/ah-django)
- Switch to the branch `ch-upgrade-django2-164859737`
- You should be able to see the upgraded django version on the `requirements.txt` file

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#164859737](https://www.pivotaltracker.com/story/show/164859737)

#### Screenshots
